src/server/components/lists/searches/helpers/getRelatedLinks.ts

```diff
@@ -0,0 +1,34 @@
+import { prisma } from "server/models/db/prisma-client";
+import { logger } from "server/services/logger";
+import { type List, ServiceType } from "shared/types";
+
+export async function getRelatedLinks( countryName: string, serviceType: ServiceType ) {
+  const list = await prisma.list.findFirst({
+    where: {
+      country: {
+        name: countryName,
+      },
+      type: serviceType
+    },
+    select: {
+      id: true,
+      jsonData: true,
+    },
+  });
+
+  if (!list) {
+    logger.info(`getRelatedLinks for ${countryName} ${serviceType}, list not found`);
+    return []
+  }
+
+  const { jsonData } = list as List;
+  const { relatedLinks = [] } = jsonData;
```

Comment:
```suggestion
  const {  relatedLinks = [] } = (list as List).jsonData;
```
Very optional suggestion but what are your thoughts on writing the code like this. One less line of code 🤷

Reply:
`(list as List).jsonData` is clunky imo, which is why I didn't go for it. for 1 loc I'm not too bothered. I wish there was a nicer way of typing prisma json objects without having to cast `as` 😭